src/commcare_cloud/ansible/setup_security_updates.yml

```diff
@@ -0,0 +1,18 @@
+---
+- hosts: all
+  become: yes
+  roles:
+   - security_upgrades
```

Comment:
You need to add `- src: weareinteractive.apt` to requirements.yml. Also this should be `weareinteractive.apt` I think